### PR TITLE
Document known Puppet reports without origin issue

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Review by maximiliankolb — APPROVED** [review]
Change looks reasonable to me.

File: guides/doc-Release_Notes/topics/foreman.adoc [hunk 1]
```diff
@@ -49,4 +49,9 @@ It is safe to remove the file or remove all content.
 
 * The `foreman_chef` plugin is unmaintained and broken since a few releases.
   It has been removed from the Installer and Packaging.
-* Google compute resource has been extracted to the https://github.com/theforeman/foreman_google[Foreman Google] plugin. {ProjectServer}s with Google compute resources enabled are automatically migrated to the plugin.
+* Google compute resource has been extracted to the https://github.com/theforeman/foreman_google[Foreman Google] plugin.
+  {ProjectServer}s with Google compute resources enabled are automatically migrated to the plugin.
+* Puppet reports https://projects.theforeman.org/issues/35684[skips Applied catalog in x.y second logs].
```
Comment (adamlazik1) [inline guides/doc-Release_Notes/topics/foreman.adoc:54]:
```suggestion
* Puppet reports https://projects.theforeman.org/issues/35684[skip "Applied catalog in x.y second logs" messages].
```

File: guides/doc-Release_Notes/topics/foreman.adoc [hunk 2]
```diff
@@ -49,4 +49,9 @@ It is safe to remove the file or remove all content.
...
+  Uneventful Puppet runs are now completely empty, which reduces database storage requirements.
+  This exposed a bug where https://projects.theforeman.org/issues/35833[Foreman is unable to determine the origin].
+  Because of this, uneventful Puppet reports will not have an origin.
```
Comment (adamlazik1) [inline guides/doc-Release_Notes/topics/foreman.adoc:57]:
```suggestion
  Because of this, uneventful Puppet reports do not have an origin.
```

**Review by adamlazik1 — APPROVED** [review]
LGTM!
Made some suggestions on wording improvements but consider them non-blocking.

**Pull request merged**